src/sum.js

```diff
@@ -6,12 +6,19 @@
  * @returns {number} The sum
  */
 function sum(array) {
-  // This is your job. :)
+  let count = 0
+
+  for(let element in array) {
+count+=array[element];
+  }
+  return count;
 }
 
 if (require.main === module) {
   console.log('Running sanity checks for sum:');
-
+console.log('-------');
+let array = [4,5,6,2,4,6,7];
+console.log(sum(array));
```

Comment:
You'll want more sanity checks that this. Does it return the correct value for an empty array, for example? How could someone looking at the output know whether the sanity checks were passing or not without having to look at the code itself?